src/proxy/http/profiles.rs

```diff
@@ -294,50 +311,69 @@ pub mod router {
         }
     }
 
+    type ConcreteRouter<T, Svc, B> =
```

Comment:
Nit: It seems like we usually put type aliases at the top of the module, so could we put this above `fn layer()`?